### PR TITLE
Hidden Power Glitch Fixed

### PR DESCRIPTION
**Pull request by Tigerball890**
OK @Zarel, I ran this with npm test and no errors, tested it out a few times, and all went well. Close it anyway for another reason if you have to, but the tests went well. 

(This is the Z-Move Hidden Power code)

### CODE REVIEW

**Comment by Tigerball890** [comment]
Someone still needs to fix this in the client

**Comment by Tigerball890** [comment]
Not the baneful bunker thing, the original pullreq

**Comment by Zarel** [comment]
Thanks for the contribution! I'll need to investigate since the problem should most likely be fixed somewhere else, though.

File: data/moves.js [hunk 1]
```diff
@@ -2584,7 +2584,7 @@ exports.BattleMovedex = {
 		accuracy: true,
 		basePower: 0,
 		category: "Status",
-		desc: "The user uses the last move used by any Pokemon, including itself. Fails if no move has been used, or if the last move used was Assist, Belch, Bestow, Chatter, Circle Throw, Copycat, Counter, Covet, Destiny Bond, Detect, Dragon Tail, Endure, Feint, Focus Punch, Follow Me, Helping Hand, Hold Hands, King's Shield, Mat Block, Me First, Metronome, Mimic, Mirror Coat, Mirror Move, Nature Power, Protect, Rage Powder, Roar, Sketch, Sleep Talk, Snatch, Spiky Shield, Struggle, Switcheroo, Thief, Transform, Trick, or Whirlwind.",
+		desc: "The user uses the last move used by any Pokemon, including itself. Fails if no move has been used, or if the last move used was Assist, Belch, Bestow, Baneful Bunker, Chatter, Circle Throw, Copycat, Counter, Covet, Destiny Bond, Detect, Dragon Tail, Endure, Feint, Focus Punch, Follow Me, Helping Hand, Hold Hands, King's Shield, Mat Block, Me First, Metronome, Mimic, Mirror Coat, Mirror Move, Nature Power, Protect, Rage Powder, Roar, Sketch, Sleep Talk, Snatch, Spiky Shield, Struggle, Switcheroo, Thief, Transform, Trick, or Whirlwind.",
```
Comment (urkerab) [inline data/moves.js]: You should update `mods/gen6/moves.js` so that Copycat has its original description.

Comment (Tigerball890) [inline data/moves.js]: Ok, didn't think about that

**Comment by Tigerball890** [comment]
Reverting in a second

**Comment by Zarel** [comment]
I didn't have time to review this from now, but I just tested this.

This was never broken, there's no need to fix it.

In the future, please test things before you try to fix them.

**Pull request closed without merging**